apps/nova/src/pages/Call.tsx

```diff
@@ -190,6 +195,7 @@ const Call: React.FC = () => {
   return (
     <div className="flex h-[calc(100vh-4rem)] overflow-hidden w-full">
       <div
+        ref={ref}
```

Comment:
The `ref` must have an ID to be able to compare it to the `document.fullscreenElement.id`
```suggestion
      <div
        ref={ref}
        id="call-page"
```